release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -203,6 +203,20 @@ Release Day
 - Every issue in the milestone is considered release blocking.
 - If you have to push the release date back, try to avoid Friday since it makes release publicity extremely difficult. Also, people seem to have patience with delay as long as the reasons are clear and openly communicated. This is your duty. You must over-communicate and ensure the team is also talking to their stakeholders (CNCF, community, press, etc.)
 - Make release day as fun as you can for the team. Plan ahead for this and do something nice.
+- The following final actions should be ordered, with successful completion of each being the entry criteria to the next.
+  - Release day morning:
+    - Go / No-Go: should generally be clear a day or three ahead of release, but the day's burndown provides a final opportunity for the team to affirm things are ready.
+    - Release notes draft merges.
+    - Branch manager does mock release build.
+    - Branch manager does mock publication.  Validate announce email.
+    - Branch manager does nomock release build.
+  - Starting approximately 4pm Pacific:
+    - Communications begins staging blog post.
+    - Branch manager does nomock publication.
+    - Request RPMs/Debs build.
```

Comment:
should we indicate it needs a Googler or link to https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/branch-manager#debs-and-rpms?